pkg/snet/arclient/client.go

```diff
@@ -333,24 +262,32 @@ type RemoteVisor struct {
 	Addr string
 }
 
-func (c *client) BindSTCPH(ctx context.Context, dialCh <-chan cipher.PubKey) (<-chan RemoteVisor, error) {
-	if c.stcphAddrCh == nil {
-		if err := c.initSTCPH(ctx, dialCh); err != nil {
+func (c *client) BindSUDPH(ctx context.Context, filter *pfilter.PacketFilter) (<-chan RemoteVisor, error) {
+	if c.sudphAddrCh == nil {
+		if err := c.initSUDPH(ctx, filter); err != nil {
 			return nil, err
 		}
 	}
 
-	return c.stcphAddrCh, nil
+	return c.sudphAddrCh, nil
 }
 
-// TODO(nkryuchkov): Ensure this works correctly with closed channels and connections.
-func (c *client) initSTCPH(ctx context.Context, dialCh <-chan cipher.PubKey) error {
-	conn, err := c.Websocket(ctx, wsPath)
+func (c *client) initSUDPH(_ context.Context, filter *pfilter.PacketFilter) error {
```

Comment:
do we need this `context.Context` arg here at all?

Reply:
We don't